BaseTools/Plugin/HostBasedUnitTestRunner/HostBasedUnitTestRunner.py

```diff
@@ -272,13 +272,25 @@ def gen_code_coverage_clang(self, thebuilder):
             if ret != 0:
                 logging.error("UnitTest Coverage: Failed generate filtered coverage XML.")
                 return 1
-        # Merge all XML files
+        # Merge package coverage XML files
         ret = self.merge_cobertura_xml_files([f"{testFile}.coverage.xml" for testFile in testList], mergedCoverageXml)
+        if ret != 0:
+            logging.error("UnitTest Coverage: Failed to merge package coverage XML files.")
+            return 1
+
+        workspace = thebuilder.env.GetValue("WORKSPACE")
+        allCoverageXml = os.path.join(workspace, 'Build', 'coverage.xml')
+        testCoverageList = glob.glob(os.path.join(workspace, 'Build', "**", "*.coverage.xml"), recursive=True)

```

Comment:
The gcc coverage above removes Build/coverage.xml if it exists before generated the merged coverage file.  This logic will overwrite the file if it already exists.  Probably safe to follow the gcc approach in case there is an error in merge_cobertura_xml_files() and the previous merged coverage file is still present.